lesson03/assignment/index.js

```diff
@@ -0,0 +1,111 @@
+var arr = []
+
+/*  
+ * Collects data from external JSON file and stores it in an array
+ * 
+ */
+function buildData() {

```

Comment:
This is an interesting approach using d3, but for our purposes you can go ahead and just copy/paste the JSON data into your JS file as a variable.

Reply:
Ignored due to criteria 1, "Used local JSON for data, not hardcoded"

Reply:
Adam, feel free to bring the data into the JS file however you'd like, but this currently does not function as-is. Check your console when running gulp and you'll see an issue with your buildData function.

Reply:
Demo video included. Data loads normally during testing

Reply:
Hi Adam, this branch/pull request's last commit was 7/16, did you add the video to a different branch? I see you mentioned that files were updated in other comments as well, but I'm not seeing them here. Can you please make sure you've pushed the updated files to this branch or point me to a branch in your fork with the updated files?

Reply:
Thanks Adam, I'm seeing the updates now. It looks like the d3 issue was with the trailing comma in the JSON data, now it's working on my end.